### PR TITLE
Move vip-import-sql to the CLI command line output

### PR DESCRIPTION
**Pull request by markowsiak**
## Description

Wires up the automated sql import command with a description and usage example.

## Steps to Test

1. Check out PR on your sandbox and `npm run build`
2. `node ./dist/bin/vip-import.js`
3. `node ./dist/bin/vip-import-sql.js`

Note that there is descriptive output.

---
_This PR was started by: [git wf pr](https://github.com/groupon/git-workflow/releases/tag/v1.3.2)_

### CODE REVIEW

File: src/bin/vip-import.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+/**
+ * Internal dependencies
+ */
+import command from 'lib/cli/command';
+import { trackEvent } from 'lib/tracker';
+
+command( )
+	.command( 'sql', 'Import SQL to your database from a file' )
+	.command( 'validate sql', 'Validate your SQL dump' )
+	.command( 'validate files', 'Validate your media file library' )
```
Comment (mjangda) [inline src/bin/vip-import.js]: Looks like these aren't working:

```
$ node ./dist/bin/vip import validate sql
...
Error: `validate sql` is not a valid subcommand. See `vip --help`
```

and

```
$ node ./dist/bin/vip import validate files
...
Error: `validate files` is not a valid subcommand. See `vip --help`
```

Comment (mjangda) [reply src/bin/vip-import.js]: Possible fix in b90c569

**Review by elazzabi — APPROVED** [review]
Tested this locally after @mjangda's fix. Works fine for me 🎉

**Review by jblz — APPROVED** [review]
lgtm.  🚀

**Pull request merged**